26-register_publication/1625-creator_navigates_to_the_reference_tab_and_selects_reference_subtype_corrigendum.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Creator selects Reference type Contribution to journal and selects subtype Corrigendum
+
+    Scenario: Creator can see fields related to Reference subtype "Corrigendum"
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Contribution to journal"
+        And they select the Reference subtype "Corrigendum"
+        Then they see fields:
+            | Search box for "Journal article" |
+            | DOI                              |
+            | Volume                           |
+            | Issue                            |
+            | Pages from                       |
+            | Pages to                         |
+            | Article number                   |
+        And they see a disabled field for Journal based on selected Journal article
+
+    Scenario: Creator see that fields for "Corrigendum" subtype are validated
```

Comment:
```suggestion
    Scenario: Creator sees mandatory fields for "Corrigendum" subtype
```

Reply:
nja, vi vil teste at feltet er synlig i forrige scenario, men nå er vi bare opptatt av at man kan se validering/feilmeldinger

Reply:
egentlig gjelder det optional fields også, om man feks skriver "ost" i felt som venter tall, men vet ikke hvordan vi evt skal dekke alle muligheter...

Reply:
^ Har lagt til mer detaljer for validering nå